proposals/0293-vehicle-type-filter.md

```diff
@@ -388,30 +420,26 @@ The below example shows valid vehicle type filters.
-
-![Sequence Diagram](../assets/proposals/0293-vehicle-type-filter/ios.png)
+1. The proxy will receive vehicle type info in `StartServiceAck` protocol message. 
+2. On receiving vehicle type information, proxy will check `SDLSupportedVehicleTypes` keys to check supported vehicle types.
+3. If the vehicle type is supported, the proxy should also notify the app about the connected vehicle type so that the app can configure SDL as required. 
+4. If the proxy determines that connected vehicle type is not supported, the proxy will end the RPC session. The application will not register on SDL enabled IVI system.
+5. If the vehicle type information is not available in `StartServiceAck` protocol message, the SDL proxy will continue with the app registration and it will rely on vehicle type information received in the `RegisterAppInterface` response. If vehicle type is not supported, the exclusive apps will be allowed to unregister from the SDL enabled system.
 
 ## Potential downsides
 
-The Android Router Service after transport connection needs to start the RPC service. If the Android Router Service does not start RPC service and relies on the client to send the `GetVehicleType` protocol message, the connected client would start another foreground service. This would force the connected app to show two notifications to users while determining vehicle type information and would need to terminate services if the app does not support the SDL enabled IVI system. 
+The SDL device Listener after transport connection needs to start the RPC service. If the SDL device Listener does not start RPC service and relies on the client for vehicle type details, the connected client would start another foreground service. This would force the connected app to show two notifications to users while determining vehicle type information and would need to terminate services if the app does not support the SDL enabled IVI system. 
 
 ## Impact on existing code
 
 Above mentioned changes need to be implemented in SDL Core, the Java Suite proxy, and the iOS proxy libraries.
 
 ## Alternatives considered
 
-### GetVehicleType as an RPC instead of Protocol message
+### Using RPC instead of Protocol message to share vehicle type info
 1. As per the current implementation, the RAI is the first RPC sent to an SDL enabled IVI system and the vehicle type information is available to the app in its response. 
-2. If `GetVehicleType` is an RPC, this behavior needs to be changed. The app should be able to use the `GetVehicleType` RPC before the `RegisterAppInterface` RPC.
-3. Since the RPC message version is sent to the app in `RAI response`, the app, when connected to an older SDL enabled IVI, would not know if the `GetVehicleType` RPC is supported or not.
+2. If `GetVehicleType` is an RPC to receive the vehicle type info, it would be the first RPC to be used. Since, the app should be able to use the `GetVehicleType` RPC to know vehicle type info before the app registration on IVI.
```

Comment:
I'm having trouble following this line. Can you please re-word? If you could let me know which requested revision required this change, that may also be helpful in clarifying this information.

Reply:
I rephrased it in commit https://github.com/smartdevicelink/sdl_evolution/pull/1067/commits/47374a485dde2f5e82fad4c5bdab8b59669c65af

Reply:
Thanks! Can you please also advise which requested revision (from this comment: https://github.com/smartdevicelink/sdl_evolution/issues/971#issuecomment-662509171) required this change?

Reply:
In the earlier proposal, we proposed "GetVehicleType" as a protocol message control frame. Hence, the alternatives considered was a comparison between the protocol message and RPC. In this revision as per Item 2, we removed a protocol message, that's why added some clarification about the purpose of RPC.